lib/components/godoc-panel-view.js

```diff
@@ -0,0 +1,39 @@
+/** @babel */
+/** @jsx etch.dom */
+
+import etch from 'etch'
+
+export default class GodocPanelView {
```

Comment:
They are talking about the `etch` package and why they don't want to add a superclass over there. But having one here is ok. It is already used by the other panel views so it just feels weird to not use it for this one too